opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -45,7 +47,7 @@ public void beforeTest() throws Exception {
         mockTracer = new MockTracer(MockTracer.Propagator.TEXT_MAP);
 
         ServletContextHandler servletContext = new ServletContextHandler();
-        servletContext.setContextPath("/");
+        servletContext.setContextPath(contextPath);
```

Comment:
It's just for tests, make it more generic and test if skip pattern works with contextPath